cmd/kosli/docs.go

```diff
@@ -180,6 +181,9 @@ func KosliGenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(st
 		// Note: The contents of the title lines could also contain < and > characters which will
 		// be lost if simply embedded in a md ## section.
 		buf.WriteString("## Examples Use Cases\n\n")
+		buf.WriteString("These examples all assume the `KOSLI_API_TOKEN`, `KOSLI_ORG`, `KOSLI_HOST`, `KOSLI_FLOW`, and `KOSLI_TRAIL`\n")
```

Comment:
in general, you don't need to use env vars. So maybe focus on the flags being set here instead of how they could/should be set?

Reply:
IIUC you mean that the wording here shouldn't mention env-vars. I agree but I can't think of exactly to reword this.
Do you have a rough suggestion?

Reply:
how about:
```
These examples all assume that the flags  `--api-token`, `--org`, `--host`, `--flow`, and `--trail` are set/provided.
```
to replace the 3 lines

Reply:
Nice.